3.rb

```diff
@@ -0,0 +1,8 @@
+ARR1 = [1,2,3,4,"X","Miles"]
```

Comment:
Why ARR1 is in capital is this global variable ?

Please read concept of variables